lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -8,7 +8,6 @@
 import json
 import itertools
 import unittest
-import pytz
```

Comment:
This file is now (almost) completely unforked.
The only difference is for one of the query counts below:
`*itertools.product(((40, 4, True), (40, 4, False)), (True, False))`